README.md

```diff
@@ -29,6 +29,7 @@ Table of Contents
 * [**eco**](https://github.com/thatvegandev/gh-eco) - Extension to explore github ecosystem, like github profile, etc.
 * [**gei**](https://github.com/github/gh-gei) - Extension to help you move your enterprise to GitHub Enterprise Cloud.
 * [**get-asset**](https://github.com/chmouel/gh-get-asset) - Fetch an asset (ie: binary) from a GitHub release.
+* [**github-repo**](https://github.com/sarumaj/gh-gr) - Pull, push and check status on multiple GitHub repositories at once.
```

Comment:
sorry for late response, this repo mantainer is kinda busy right now :smiling_face_with_tear: 
Thanks for PR, great cli indeed. Btw, can you remove the `github` prefix? so it would be `repo` or `gr` so it will be aligned with the rest. And please reorder the list after rename.